pkg/apis/build/v1alpha2/cosign_types.go

```diff
@@ -0,0 +1,13 @@
+package v1alpha2
+
+// +k8s:openapi-gen=true
+type CosignConfig struct {
+	// +listType
+	Annotations []CosignAnnotation `json:"annotations"`
```

Comment:
I think `Annotations` and the `CosignAnnotation.Name` and `CosignAnnotation.Value` can be `omitempty` ?

Reply:
done, resolving